zulipterminal/ui_tools/views.py

```diff
@@ -1519,3 +1519,92 @@ def keypress(self, size: urwid_Size, key: str) -> str:
             )
             return key
         return super().keypress(size, key)
+
+
+class FullMsgView(urwid.Frame):
+    def __init__(self, controller: Any, message: Message,
+                 topic_links: 'OrderedDict[str, Tuple[str, int, bool]]',
+                 message_links: 'OrderedDict[str, Tuple[str, int, bool]]',
+                 time_mentions: List[Tuple[str, str]],
+                 title: str) -> None:
+        self.title = title
+        self.controller = controller
+        self.message = message
+        self.topic_links = topic_links
+        self.message_links = message_links
+        self.time_mentions = time_mentions
+        self.max_cols, self.max_rows = controller.maximum_popup_dimensions()
+
+        # Get rendered message
+        msg_box = MessageBox(message, controller.model, None)
+        if not msg_box.need_recipient_header():
+            msg_box.widget_list.insert(0, msg_box.top_header_bar(msg_box))
+        msg_w = msg_box.widget_list
+
+        header = msg_w[:2] + [urwid.Divider()]
+        _content_formatted = urwid.Pile([msg_w[2].original_widget[-1][1]])
+
+        # Get raw message content
+        _content_raw = controller.model.fetch_raw_message(message['id'])
+        _content_raw_w = urwid.Pile([urwid.Text(_content_raw)])
+
+        tab_list = [
+            {'label': 'Raw content',       'content': _content_raw_w},
```

Comment:
The label in the Message Information view says "Full message" but this renders "Raw content" first. I think we should show the "Raw content" later if we end up going the _tabs_ way.

Reply:
Also, it is not evident whether the user is being presented with "Full message" or "Raw content" in the UI.